src/proof/er/er_proof.h

```diff
@@ -119,8 +118,10 @@ class ErProof
    * @param usedClauses The CNF formula that we're deriving bottom from.
    * @param dratBinary  The DRAT proof from the SAT solver, as a binary stream.
    */
-  static ErProof fromBinaryDratProof(const ClauseUseRecord& usedClauses,
-                                     const std::string& dratBinary);
+  static ErProof fromBinaryDratProof(
+      const std::map<ClauseId, prop::SatClause>& clauses,
```

Comment:
This is mostly a matter of taste but I have a preference for using `std::unordered_map` when order doesn't matter. Performance-wise, the best pick depends a bit on what your data looks like on average but for larger maps, `std::unordered_map` should perform better.

Reply:
You're right. `unordered_map` is better for this DS.